### PR TITLE
Autodesk,Inc.AutodeskDownloadManager version 4.0.19.0

### PR DESCRIPTION
**Pull request by oliveirahsi**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59691)

### CODE REVIEW

File: manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager
+PackageVersion: 4.0.19.0
+PackageLocale: en-US
+Publisher: Autodesk, Inc.
+PackageName: Autodesk Download Manager
+License: MIT
+Copyright: Copyright 2015 Autodesk, Inc. All rights reserved.
+ShortDescription: Autodesk Download Manager
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.locale.en-US.yaml:11]: Please provide a better description

File: manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.locale.en-US.yaml:4]: The publisher already exists as `Autodesk` so the package ID here should be updated to match

File: manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager
+PackageVersion: 4.0.19.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.installer.yaml:8]: Package does not install silently with the current Installer Type; Silent installation switches must be provided

File: manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager
+PackageVersion: 4.0.19.0
+PackageLocale: en-US
+Publisher: Autodesk, Inc.
+PackageName: Autodesk Download Manager
+License: MIT
```
Comment (Trenly) [inline manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.locale.en-US.yaml:9]: Autodesk uses a proprietary EULA for the software
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm stuck here:
image

**Review by Trenly — CHANGES_REQUESTED** [review]
The URL provided seems to be for Autodesk.DWGTrueView.2023 and not the AutodeskDownloadManager

**Pull request closed without merging**